src/Compilers/CSharp/Portable/Binder/SwitchBinder.cs

```diff
@@ -181,66 +200,35 @@ internal override bool IsLabelsScopeBinder
             }
         }
 
-        private void BuildSwitchLabels(SyntaxList<SwitchLabelSyntax> labelsSyntax, Binder sectionBinder, ref ArrayBuilder<LabelSymbol> labels)
+        private void BuildSwitchLabels(TypeSymbol switchGoverningType, SyntaxList<SwitchLabelSyntax> labelsSyntax, Binder sectionBinder, ArrayBuilder<LabelSymbol> labels, DiagnosticBag tempDiagnosticBag)
```

Comment:
I believe this is an artifact of an incomplete refactoring. I will check and, if so, remove the parameter.